tests/unit/catalogue/test_product_attributes.py

```diff
@@ -332,6 +333,32 @@ def test_entity_attributes(self):
             # pylint: disable=pointless-statement
             product.attr.entity
 
+    def test_can_be_pickled_when_initialized(self):
+        """Should be able to pickle and unpickle an initialized ProductAttributesContainer"""
+        product = Product.objects.get(pk=self.product.pk)
+
+        self.assertEqual(product.attr.weight, 3)
+        self.assertTrue(product.attr.initialized)
+
+        pickled_attrs = pickle.dumps(product.attr)
+        print(pickled_attrs)
```

Comment:
I think this has been left here by accident?

Reply:
Fixed